service/storage_service.cc

```diff
@@ -778,14 +778,14 @@ class topology_coordinator {
     };
 
     future<> handle_ring_transition(node_to_work_on&& node) {
-        bool res;
+        bool exec_command_res;
```

Comment:
What is wrong with less descriptive name? Will we have another `res` here in the future patches?